test/torchaudio_unittest/transforms/batch_consistency_test.py

```diff
@@ -33,13 +33,14 @@ def test_batch_Resample(self):
         self.assertEqual(computed, expected)
 
     def test_batch_MelScale(self):
-        specgram = torch.randn(2, 31, 2786)
+        n_stft = 31
+        specgram = torch.randn(2, n_stft, 2786)
 
         # Single then transform then batch
-        expected = torchaudio.transforms.MelScale()(specgram).repeat(3, 1, 1, 1)
+        expected = torchaudio.transforms.MelScale(n_stft=n_stft)(specgram).repeat(3, 1, 1, 1)
```

Comment:
Note: (follow-up, not the scope of this PR) For batch consistency test, use of repeated does not catch some edge case. https://github.com/pytorch/audio/pull/1315
We should update the logic at some point.